decko/lib/decko/generators/decko/templates/Gemfile

```diff
@@ -1,18 +1,21 @@
 source 'http://rubygems.org'
 
 <% if @gem_path.present? %>
-gem 'decko',    :path=>"<%= @gemfile_gem_path %>/decko"
-gem 'card',     :path=>"<%= @gemfile_gem_path %>/card", :require => false
-gem 'cardname', :path=>"<%= @gemfile_gem_path %>/cardname", :require => false
+path "<%= @gemfile_gem_path %>" do
+  gem 'decko'
+  gem 'card', :require => false
+  gem 'cardname', :require => false
+end
 <% else %>
 gem 'decko'
 <% end %>
 
 gem '<%= database_gemfile_entry.name %>'<%= %(, '#{database_gemfile_entry.version}') if database_gemfile_entry.version %>
 
-# Use Unicorn or Thin as server
+# Use Thin or Unicorn as server
```

Comment:
if server not specified, it will use webrick, yes?

I should probably add some copy here to refer to our web docs on hosting in production

Reply:
yes. webrick gives a lot of `content-length` warning messages when it serves files. thin doesn't care about that. That's why I switched.